In app update changelog (#1984)
microsoft/appcenter-sdk-apple: CHANGELOG.md

> ## Version 3.0.1 (Under development)

Reviewer: ```suggestion
## Version 3.1.0 (Under development)
```

### App Center Distribute

> * **[Feature]** Added a `DisableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update. 

Reviewer 2: Somehow we are using present tense for CHANGELOG.

Reviewer 2: ```suggestion
* **[Feature]** Add a `disableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update. 
```

> * **[Feature]** Added a `CheckForUpdate` API to manually check for an update.

Reviewer 2: ```suggestion
* **[Feature]** Add a `checkForUpdate` API to manually check for update.
```

## Version 3.0.1 (Under development)

### App Center Distribute

> * **[Feature]** Add `disableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update and a `checkForUpdate` API for a manual update check.

Reviewer 3: ```suggestion
* **[Feature]** Added a `DisableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update. 
* **[Feature]** Added a `CheckForUpdate` API to manually check for an update.
```
